src/Routes/Solar.elm

```diff
@@ -12,136 +13,120 @@ solar model =
-            , li [ class "mercury" ]
+            , li [ class <| "mercury" ++ (getPlanetBlurClass model PeersFamily) ]
                 [ a [ href "#mercury" ]
                     [ span []
-                        [ text "Mercury" ]
+                        []
                     ]
                 ]
-            , li [ class "venus border" ]
+            , li [ class <| "venus border" ++ (getPlanetBlurClass model PeersFamily) ]
                 [ a [ href "#venus" ]
                     [ span []
-                        [ text "Venus" ]
+                        []
                     ]
                 ]
-            , li [ class "earth" ]
-                [ a [ href "#earth" ]
-                    [ span []
-                        [ text "Earth"
-                        , span [ class "moon" ]
-                            [ text "& Moon" ]
-                        ]
+            , li [ class <| "earth" ++ (getPlanetBlurClass model SchoolsCommunity), onClick (FocusOrganisation ChildrenServices) ]
+                [ span []
+                    [ span [ class "moon" ]
+                        []
                     ]
-                ]
-            , li [ class "mars" ]
-                [ a [ href "#mars" ]
-                    [ span []
-                        [ text "Mars" ]
+                , div []
+                    [ div []
+                        (if model.focusedOrganisation == Just ChildrenServices then
+                            [ h4 [ class "f4" ] [ text "Barnet Children and Young People Plan" ]
+                            , p [ class "f5" ] [ text "A partnership with children, young people and different organisations that has produced a new child-friendly plan for 2016 - 2020 that reflects the priorities, needs and aspirations of the local population." ]
+                            ]
+                         else
+                            []
+                        )
                     ]
                 ]
-            , li [ class "asteroids_meteorids" ]
+            , li [ class <| "mars" ++ (getPlanetBlurClass model SchoolsCommunity) ]
                 [ span []
-                    [ text "Asteroids & Meteorids" ]
+                    []
                 ]
-            , li [ class "jupiter border" ]
-                [ a [ href "#jupiter" ]
-                    [ span []
-                        [ text "Jupiter" ]
-                    ]
+            , li [ class <| "asteroids_meteorids" ++ (getPlanetBlurClass model SchoolsCommunity) ]
+                [ span []
+                    []
                 ]
-            , li [ class "saturn" ]
-                [ a [ href "#saturn" ]
-                    [ span []
-                        [ text "Saturn & "
-                        , span [ class "ring" ]
-                            [ text "Ring" ]
+            , li [ class <| "jupiter border" ++ (getPlanetBlurClass model HealthHousingEmployabilitySafety), onClick (FocusOrganisation SocialCareDirect) ]
```

Comment:
When I have lots of classes I need to add to an element I often find it cleaner and more flexible to define a helper like this:

```elm
classes : List String -> Attribute msg
classes cxs = class <| String.join " " cxs
```

```elm
venus : Model -> Html msg
venus model =
    li
        [ classes
            [ "venus"
            , "border"
            , getPlanetBlurClass model PeersFamily
            ]
        ]
        [ a [ href "#venus" ] [ span [] [] ]
        ]
```
That way you don't have to worry about putting spaces around strings to make them valid css classes like you have here:

```elm
Just focussedGroup ->
    if orgGroup == focussedGroup then
        ""
    else
        " blur "
```

and less brackets & (++)

Reply:
@andrewMacmurray amazing!!! ❤️  Ill make these changes then and push it...and leave to @astroash to change the model...as she was the one that designed the amazing boilerplate! 😍

Reply:
@rebecacalvoquintero I'd make the changes here too so you can get a feel for them (rather than waiting for the next project). You can always make a PRs to the boilerplate too (beauty of open source 😉 )